Automattic/jetpack
# Contact Form: Update Form child blocks names

Fixes #27092

#### Changes proposed in this Pull Request:
* Update Form child blocks names to be more meaningful, as they'll now appear as standalone blocks

New names can be found here: pcsBup-jU-p2

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Create a Page/Post and open the Block Library
* Under the Forms section, you should see the Form child blocks with updated names.
